### DISCUSSION TITLE
Date labels to custom text

### DISCUSSION THREAD

**Discussion by chrisxyg** [thellmund/Android-Week-View#237] [General]
Hi and thank you for this library!!!
The library works really nice, but i would like to ask if we can change the date labels to custom texts?

Thank you!

**Comment by thellmund** [comment]
Hi 👋 What exactly do you mean by “custom texts”? Do you want to be able to provide `SpannableString` and alike?

**Reply by chrisxyg** [reply]
Hi, thank you for your response
I am sorry, I wrote "custom texts" by mistake. I meant "custom views". 
But yes, for now i am using SpannableString ...
